src/compiler/crystal/tools/doc/generator.cr

```diff
@@ -283,6 +283,8 @@ class Crystal::Doc::Generator
 
     github_remote_pattern = /github\.com(?:\:|\/)((?:\w|-|_)+)\/((?:\w|-|_|\.)+)/
     github_remotes = remotes.lines.select &.match(github_remote_pattern)
+    @is_crystal_repo = github_remotes.any? { |gr| gr =~ %r{github\.com[/:]crystal-lang/crystal} }
```

Comment:
Not having to espace forward slashes is why I use `%r` over `/.../` in the first place.